openstack/03b-IPv6/icmpv6rpl.c

```diff
@@ -567,3 +567,21 @@ void sendDAO() {
       openqueue_freePacketBuffer(msg);
    }
 }
+
+void icmpv6rpl_setDIOPeriod(uint16_t dioPeriod){
+    icmpv6rpl_vars.periodDIO = dioPeriod/5; // delayDIO will skip 4 times in 5 dio period

```

Comment:
Fixed by call DIO or DAO every `dioPeriod` and `daoPeriod`, without `delayDIO` or `delayDAO`